packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json

```diff
@@ -0,0 +1,1125 @@
+            "filtered": "__count__ hidden"
+        },
+        "context": {
+            "name": "Contexto dos Dados",
+            "label": "contexto",
+            "none": "nó selecioado",
+            "refresh": "atualize para carregar",
+            "empty": "vazio",
+            "node": "Nó",
+            "flow": "Fluxo",
+            "global": "Global",
+            "deleteConfirm": "Você tem certeza que deseja remover este item?",
+            "autoRefresh": "Atualizar na mudança de seleção",
+            "refrsh": "Atualizar",
+            "delete": "Remover"
+        },
+        "palette": {
+            "name": "Gerenciamento de paleta",
+            "label": "paleta"
+        },
+        "project": {
+            "label": "proejto",
+            "name": "Projeto",
+            "description": "Descrição",
+            "dependencies": "Dependencia",
+            "settings": "Configurações",
+            "noSummaryAvailable": "Nenhum resumo disponível",
+            "editDescription": "Editar a descrição do projeto",
+            "editDependencies": "Editar dependências do projeto",
+            "noDescriptionAvailable": "Descrição não disponível",
+            "editReadme": "Editar README.md",
+            "showProjectSettings": "Mostrar configurações do projeto",
+            "projectSettings": {
+                "title": "Configurações do Projeto",
+                "edit": "editar",
+                "none": "Nenhum",
+                "install": "installar",
+                "removeFromProject": "remover do projeto",
+                "addToProject": "adicioanr ao projeto",
+                "files": "Arquivos",
+                "flow": "Fluxos",
+                "credentials": "Credenciais",
+                "package": "Pacote",
+                "packageCreate": "O arquivo será criado quando as alterações forem salvas",
+                "fileNotExist": "Arquivo não existe",
+                "selectFile": "Selecione o arquivo",
+                "invalidEncryptionKey": "Chave de criptografia inválida",
+                "encryptionEnabled": "Criptografia habilitada",
+                "encryptionDisabled": "Criptografia desabilitada",
+                "setTheEncryptionKey": "Defina a chave de criptografia",
+                "resetTheEncryptionKey": "Reset the encryption key",
+                "changeTheEncryptionKey": "Redefina a chave de criptografia",
+                "currentKey": "Chave atual",
+                "newKey": "Nova chave",
+                "credentialsAlert": "Isso excluirá todas as credenciais existentes",
+                "versionControl": "Controle de versão",
+                "branches": "Branches",
+                "noBranches": "Sem branches",
+                "deleteConfirm": "Tem certeza de que deseja excluir o branch local '__name__'? Isto não pode ser desfeito.",
+                "unmergedConfirm": "O branch local '__name__' tem alterações não mescladas que serão perdidas. Tem certeza que deseja deletar?",
```

Comment:
Change to "O branch local '__name__' tem alterações não mescladas que serão perdidas. Tem certeza que deseja remover estas alterações?"